project-base/storefront/components/Pages/App/PageHeadScripts.tsx

```diff
@@ -0,0 +1,25 @@
+import getConfig from 'next/config';
+import Head from 'next/head';
+import { useCookiesStore } from 'store/useCookiesStore';
+import { isClient } from 'utils/isClient';
+
+export const PageHeadScripts: FC = () => {
+    const isUserSnapEnabled = useCookiesStore((store) => store.isUserSnapEnabled);
+    const { publicRuntimeConfig } = getConfig();
+    const userSnapApiKey = publicRuntimeConfig.userSnapApiKey;
+
+    return (
+        <Head>
+            {isUserSnapEnabled && isClient && userSnapApiKey && (
+                <script
```

Comment:
I'm not sure about it... isn't possible to write it somehow.... readable?

Reply:
This is written the same way as other scripts loading - for example GTM - GtmHeadScripts.tsx. So to make you more happy i tried to format the code (;